dhis-2/pom.xml

```diff
@@ -821,7 +841,8 @@
                                 <RestrictImports>
                                     <reason>Do not use this json library. Use com.google.gson instead</reason>
```

Comment:
when should one use com.google.gson? for testing we can/should advocate for https://github.com/dhis2/json-tree 😋 @jbee

Reply:
Absolutely.
We just cannot ban com.google.gson yet as it is widely used in the system